examples/customresources/models/mygizmo/mygizmo.go

```diff
@@ -15,8 +15,24 @@ import (
 // Model is the full model definition.
 var Model = resource.NewModel("acme", "demo", "mygizmo")
 
+// Config is the gizmo model's config.
+type Config struct {
+	Arg1 string `json:"arg1"`
+}
+
+// Validate checks the attribute of myGizmo to ensure that an "arg1" attribute exists
+// the model will not initialize if this is not set.
+func (cfg *Config) Validate(path string) ([]string, error) {
+	if cfg.Arg1 == "" {
+		return nil, fmt.Errorf(`expected "Arg1" attribute for myGizmo %q`, path)
```

Comment:
```suggestion
		return nil, fmt.Errorf(`expected "arg1" attribute for myGizmo %q`, path)
```
It would be lowercase in the JSON.